samples/consuming-tas-utility/src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,37 @@
+# Set root logger level to error
+#log4j.rootLogger=${app.log.root.level}, Console, File
+rootLogger.level=INFO
+rootLogger.appenderRef.stdout.ref=ConsoleAppender
+rootLogger.appenderRef.rolling.ref=RollingAppender
+
+###### Console appender definition #######
+# All outputs currently set to be a ConsoleAppender.
+# Direct log messages to stdout
+appender.ConsoleAppender.type=Console
+appender.ConsoleAppender.name=ConsoleAppender
+appender.ConsoleAppender.layout.type=PatternLayout
+# use log4j NDC to replace %x with tenant domain / username
+appender.ConsoleAppender.layout.pattern=%d{ISO8601} %x %-5p [%c{3}] [%t] %replace{%m}{[\r\n]+}{}%n
```

Comment:
Is there a reason why we're changing the layout pattern compared to the original implementation? 

In general I'd expect that we try and keep the patterns that are already present, e.g. in this case: `[%t] %d{HH:mm:ss} %-5p %c{1}:%L - %m%n` whilst replacing the newlines therefore making it: `[%t] %d{HH:mm:ss} %-5p %c{1}:%L - %replace{%m}{[\r\n]+}{}%n`

**Note: this applies to all patterns in the PR.**